gems/pending/VMwareWebService/MiqPbmInventory.rb

```diff
@@ -0,0 +1,33 @@
+require 'VMwareWebService/PbmService'
+
+module MiqPbmInventory
+  def pbm_initialize(server, username, password)
+    @pbm = PbmService.new(server, username, password) if @apiVersion >= '5.5'
+  end
+
+  def pbmProfiles
```

Comment:
nitpick: What do you think to have either this renamed as pbmProfilesByUid or return a list of profiles?